cpp/ql/test/library-tests/dataflow/dataflow-tests/dataflow-ir-consistency.expected

```diff
@@ -30,6 +30,9 @@ localCallNodes
 postIsNotPre
 postHasUniquePre
 uniquePostUpdate
+| ref.cpp:83:5:83:17 | Chi | Node has multiple PostUpdateNodes. |
+| ref.cpp:100:34:100:36 | InitializeIndirection | Node has multiple PostUpdateNodes. |
+| ref.cpp:109:5:109:22 | Chi | Node has multiple PostUpdateNodes. |
```

Comment:
I've described the source of these consistency failures [here](https://github.slack.com/archives/CP0LHP150/p1586173820201600). According to aschackmull it's not clear whether this actually causes any problems.